.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,6 +16,7 @@ Don't reference an issue in the PR title because GitHub does not support auto-li
 - [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
 - [ ] Tests created for changes (if applicable)
 - [ ] Manually tested changed features in running JabRef (always required)
+- [ ] The PR contains no or minimal AI-generated code (If any present, it is trivial and I fully understand it)
```

Comment:
What I don't like about this topic is "AI usage" != bad. It has nothing to do with goodness/effectivenes.

I like that you didn't wrote it like in that SUSE project, where no AI code is allowed (if we were to discuss this topic, then that will be too much philosophy). And because JabRef is used for teaching, we definitely need a section on AI.

But I'm not sure about that additional check. I think this is + mind effort to click that, + mind effort to think "I used copilot, is it ok or not...", "It's small, it's okay...", "No it's big, what should I do"...

I think it's best to populate some documentation page about that, where you could tell more **how to use AI correctly**.

Yes! I think the idea should be "the correct usage of AI" but not "disallowance of AI" or "to which extent you should use AI".

Reply:
Good points, and I agree.
Yes, the usage of AI is not "bad", except that a new contributor can get away with a PR addressing a good first issue merged, having learnt absolutely nothing as well as taking away the opportunity from someone who would want to work on it (and not just get a PR merged).

In case of effectiveness, we can use the same argument, pass every good first issue through AI, have them solved with minimal effort and not wait for the students to do it.

Effective use of AI and the extent - yes, I like this idea too! You can co-author this PR to change any text that you feel fit (or add to documentation/contributing guidelines).

Reply:
@InAnYan Discussion: How does this sound:
"I fully understand the working of any code picked up from any external source/generated using AI."